xds/internal/xdsclient/controller/version/v2/loadreport.go

```diff
@@ -149,5 +152,17 @@ func (v2c *client) SendLoadStatsRequest(s grpc.ClientStream, loads []*load.Data)
 
 	req := &lrspb.LoadStatsRequest{ClusterStats: clusterStats}
 	v2c.logger.Infof("lrs: sending LRS loads: %+v", pretty.ToJSON(req))
-	return stream.Send(req)
+	if err := stream.Send(req); err != nil {
+		return getStreamError(stream)
+	}
+	return nil
+}
+
+func getStreamError(stream lrsStream) error {
+	for {
```

Comment:
Should there be a way for this for loop to terminate other than when `Recv()` returns non-nil error?

Reply:
`stream.Recv` will terminate with an error if the context for the RPC is canceled, which is also how `sendLoads` exits (in `transport.go`).  So I believe this is fine.